app/uk/gov/hmrc/taxenrolmentassignmentfrontend/orchestrators/MultipleAccountsOrchestrator.scala

```diff
@@ -216,4 +217,42 @@ class MultipleAccountsOrchestrator @Inject()(
         Left(InvalidUserType(optRedirectUrl))
     }
   }
+
+  def getSAForPTAlreadyEnrolledDetails(
```

Comment:
Are there tests for this fuction?

Reply:
You're right, I will add a handful of test scenarios for this method